README.md

```diff
@@ -42,6 +62,66 @@ $ pip3 install .
 Note: On some machines, you have to use `pip` instead of `pip3` but in most cases,
 you will want to use `pip3` if it's available for your platform.
 
+For regular use, we recommend taking the archived files from [our release page](https://github.com/cyberark/conjur-api-python3/releases).
+
+### Update from Ruby to Python CLI
+
+To ensure a smooth experience moving from the Ruby to Python CLI, we recommend that you either remove the Ruby CLI from your machine or rename it.
+
+To do so:
+
+1. Run `which conjur` to get the path of the Ruby CLI gem
+2. Navigate to the outputted path and either remove it or update it `conjur.v6`
```

Comment:
what?

Reply:
We basically want to make `conjur --help` not point to the Ruby CLI, rather Python because we are calling it by the same name. To do that, they can either delete the gem or overwrite the name of the Ruby CLI with `conjur.v6` or whatever else they want to call it. Make more sense?